portfolio/src/main/webapp/events.html

```diff
@@ -27,33 +27,32 @@ <h2>Join a protest in:</h2>
         </div>
         </div>
         <div id="main">
-        <div>
-        <h2>Upcoming events</h2>
-        <div class="dropdown">
-            <button class="dropbtn">Change view</button>
-            <div class="dropdown-content">
-            <a href='#'>List</a>
-            <a href="#">Grid</a>
+            <div id="upcoming">
+            <h2>Upcoming events</h2>
             </div>
-        </div>
-            <div id="maps">
-            <div id="column-1">
+            <div id="events" class="container">
+                
+            <div class="child">
```

Comment:
Where does this show up in the page? In the screenshot you added I don't see anything that says "Map 1" for example